config/orm.js

```diff
@@ -4,7 +4,11 @@ function createQmarks(num){
     var arr=[];
 
     for (var i = 0; i < num; i++) {
-        arr.push("?");
+        if (process.env.NODE_ENV === 'production') {
+            arr.push("$" + (arr.length + 1));
+        } else {
+            arr.push("?");
+        }
```

Comment:
This code uses Postgres parameterization convention for production, which is the env for Heroku.  Otherwise, it uses mysql for local development.